### PR TITLE
fixed sql bugs and schema

### PR DESCRIPTION
**Pull request by MahinRahman8901**
Fixed the issue with the case not being available, and then sorted out the all the bash scripts aswell

### CODE REVIEW

File: database/insert_schema.sh
Comment (ErvinRex) [inline database/insert_schema.sh:1]: Great bash command, simplifying the seeding process.

File: database/schema.sql
Comment (ErvinRex) [inline database/schema.sql:1]: Great job fixing the schema sql file, making it a lot more readable.

**Comment by MahinRahman8901** [comment]
closes #83 #82 #78

**Review by ErvinRex — APPROVED** [review]
Great job simplifying these scripts, overall makes the whole team understand our database schema more clearly!

**Pull request merged**